src/System.Numerics.Vectors/src/System.Numerics.Vectors.csproj

```diff
@@ -6,13 +6,17 @@
     <RootNamespace>System.Numerics</RootNamespace>
     <DocumentationFile>$(OutputPath)$(MSBuildProjectName).xml</DocumentationFile>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <IsPartialFacadeAssembly Condition="'$(TargetGroup)'=='netfx' OR '$(TargetGroup)'=='net46'">true</IsPartialFacadeAssembly>
+    <IsNetFx Condition="'$(TargetGroup)'=='netfx' OR '$(TargetGroup)'=='net46'">true</IsNetFx>
+    <IsNetCoreApp Condition="'$(TargetGroup)'=='netcoreapp'">true</IsNetCoreApp>
```

Comment:
You have to do this for uap as well. It won't work (probably not even build) otherwise.